### PR TITLE
Implement event severity module

### PR DESCRIPTION
**Pull request by Mloweedgar**
ref https://github.com/CodeTanzania/ewea/issues/16

### CODE REVIEW

File: src/Home/index.js [hunk 1]
```diff
@@ -62,9 +62,15 @@ const routes = [
...
+    path: '/app/eventseverity',
+    icon: alertIcon,
+    description: modules.eventSeverity,
+  },
   {
     name: 'Administrative Areas',
-    path: '/app/administrativeareas',
+    path: '/app/administrativeAreas',
     icon: geographicalFeaturesIcon,
     description: modules.administrativeareas,
```
Comment (rijkerd) [inline src/Home/index.js]: Change to module.administrativeAreas

Comment (rijkerd) [inline src/Home/index.js]: Change to administrativeAreas

File: src/Events/components/EventSeverity/Form/index.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+} from '@codetanzania/ewea-api-states';
+import { Button, Form, Input } from 'antd';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { notifyError, notifySuccess } from '../../../../util';
+
+/**
+ * @class
+ * @name EventSeverityForm
+ * @description  Render form for creating a new event group
```
Comment (rijkerd) [inline src/Events/components/EventSeverity/Form/index.js]: Rename to event severity

Comment (rijkerd) [inline src/Events/components/EventSeverity/Form/index.js]: Rename to event severity

File: src/Home/index.js (hunk 1 above)
Comment (rijkerd) [inline src/Home/index.js:87]: Change to modules.administrativeAreas

**Review by rijkerd — CHANGES_REQUESTED** [review]

**Comment by Mloweedgar** [comment]
@BenMaruchu conflicts have been resolved, branch can be merged

**Review by BenMaruchu — APPROVED** [review]

**Comment by BenMaruchu** [comment]
@Mloweedgar Can you resolve these conflicts?

**Comment by Mloweedgar** [comment]
@BenMaruchu I have fixed the conflicts.

**Pull request merged**